test/supports/uv_lock_project/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+name = "uv-lock-project"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.13"
+dependencies = [
+    "modal>=1.0.3",
```

Comment:
Note that I'm still pretty unhappy about this "requirement" since the modal library dependencies do not correspond directly to the dependencies required in a Modal container.